frontend/src/app/event/event-editor/event-editor.component.html

```diff
@@ -0,0 +1,31 @@
+<form [formGroup]="eventForm" (ngSubmit)="onSubmit()" *ngIf="adminPermission$ | async">
+    <mat-card>
+        <mat-card-header>
+            <mat-card-title *ngIf="event.id == -1">Create Event</mat-card-title>
+            <mat-card-title *ngIf="event.id != -1">Update Event</mat-card-title>
```

Comment:
Hmmmmmm. I wonder if `id` should just be `undefined` in the case of a new event? That's probably more idiomatic than using `-1`